### PR TITLE
Sentry config fix for Dev env

### PR DESCRIPTION
**Pull request by SagarKarwande**
## Why?
Due to improper config for Sentry in `dev` environment. Stack trace was not logged on IEx when any error occurs in dev environment.

## This change addresses the need by:
- Fix sentry config.

[delivers #162754691]

## Checklist

- [ ] I have read [CONTRIBUTING.md][contributing].
- [ ] My code follows the [style guidelines][contributing] of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code does not generate any (new) [`credo`][credo] and compile-time warnings.
- [ ] I have updated the documentation wherever necessary.
- [ ] I have added tests to cover my changes.

[contributing]: https://github.com/aviabird/snitch/CONTRIBUTING.md
[credo]: https://github.com/rrrene/credo

### CODE REVIEW

File: apps/snitch_api/config/config.exs [hunk 1]
```diff
@@ -41,6 +41,16 @@ config :snitch_api, SnitchApi.Guardian,
 config :snitch_api, frontend_checkout_url: System.get_env("FRONTEND_CHECKOUT_URL")
 config :snitch_api, hosted_payment_url: System.get_env("HOSTED_PAYMENT_URL")
 
+config :sentry,
```
Comment (ramansah) [inline apps/snitch_api/config/config.exs]: Can we have `sentry` config in project root's config? That will prevent duplicate entries in AdminApp and SnitchApi.

Comment (SagarKarwande) [reply apps/snitch_api/config/config.exs]: Moved the Sentry config to project root config.

Comment (SagarKarwande) [reply apps/snitch_api/config/config.exs]: Also we have `sentry` dependencies in all 3 apps. Should we just keep the dependency on `core`, as `admin` and `api` app has dependency on `core`

@pkrawat1 @ramansah Any thought on this?

Comment (ramansah) [reply apps/snitch_api/config/config.exs]: Yes, that makes sense :+1:

**Pull request merged**